### DISCUSSION TITLE
Reverse proxying a remote host

### DISCUSSION THREAD

**Discussion by Silun** [nginx-proxy/nginx-proxy#2330] [Q&A]
Hello,

what is the recommended way to reverse proxy a remote host? I have a docker host running a couple of containers, and I proxy them via `VIRTUAL_HOST` and `LETSENCRYPT_HOST` environment variables so that nginx-proxy auto-generates the configuration and manages the SSL certificate as well.

Is there a way to make nginx-proxy do this for websites that are not located in docker containers on the same host? Suppose I have a service on another server on the same network, and I want to proxy this service the same as I do the local containers via `VIRTUAL_HOST` and `LETSENCRYPT_HOST`. Is there a way to achieve the same elegant automatism for such services that are not running in the same local docker instance?

I have done some searching but all I could find are custom solutions where I mount some config files into the proxy container. Are there maybe some ways to add virtual hosts to nginx-proxy via environment variables or is there maybe a dummy docker container that I can attach the `VIRTUAL_HOST` and `LETSENCRYPT_HOST` environment variables to, and it will do the proxying as described?

**Comment by buchdag** [comment]
> Is there a way to make nginx-proxy do this for websites that are not located in docker containers on the same host ?

At this time, no, this is not a feature of nginx-proxy.

It might be possible via relay containers that proxy to the remote host but I don't have any specific image or how-to to recommend.
